include/my_global.h

```diff
@@ -28,7 +28,9 @@
 #include "my_config.h"
 
 #define __STDC_LIMIT_MACROS	/* Enable C99 limit macros */
+#ifndef __STDC_FORMAT_MACROS
 #define __STDC_FORMAT_MACROS	/* Enable C99 printf format macros */
+#endif /* __STDC_FORMAT_MACROS */
 #define _USE_MATH_DEFINES       /* Get access to M_PI, M_E, etc. in math.h */
```

Comment:
I see why, PerconaFT sets it in TokuSetupCompiler.cmake and that conflicts with the my_global.h definition, but I think the problem is not the server code but PerconaFT and probably best addressed there if possible. Even if allowed in my_global.h, shouldn't all __STDC definitions here get the same treatment to prevent other/future issues?